src/functionalTest/java/uk/gov/hmcts/reform/pip/subscription/management/utils/OAuthClient.java

```diff
@@ -13,9 +13,18 @@ public class OAuthClient {
     @Value("${CLIENT_SECRET_FT}")
     private String clientSecret;
 
+    @Value("${CLIENT_ID}")
```

Comment:
I don't think you need the two new clientIdDataManagement + secret variables here, the existing variables above should work

Reply:
Removed :)